### ISSUE TITLE
<dao> elements with dead links should not be indexed into Solr fields

### ISSUE THREAD

**Issue by jrgriffiniii** [pulibrary/pulfalight#3]
By default, ArcLight assumes that all <dao> elements link to digital surrogates for items described in the finding aid.  This is not true for all cases in the PULFA finding aid EAD Documents, as some reference PDF's:
```
pulfa/eads/eng/PPL001.EAD.xml:                        <dao xlink:href="http://diglib.princeton.edu/pdfs/PPL001/c0037.pdf" xlink:type="simple"/>
```
...some reference ARKs:
```
pulfa/eads/mss/WC064.EAD.xml:                    <dao xlink:href="http://arks.princeton.edu/ark:/88435/ns064676j" xlink:type="simple"/>
```
...some reference METS Documents:
```
pulfa/eads/mss/WC116.EAD.xml:            <dao xlink:type="simple" xlink:role="http://www.loc.gov/METS/" xlink:href="http://findingaids.princeton.edu/folders/WC116/c01.mets"/>
```
...some reference static web content (presumably used for the design of previous finding aid platforms):
```
pulfa/eads/mss/WC092.EAD.xml:        <dao xlink:title="Princeton University Manuscripts Division" xlink:href="bioghist-images/msslogo.jpg" xlink:type="simple"/>
```
...and some reference other web sites:
```
pulfa/eads/mudd/univarchives/AC190.EAD.xml:        <dao xlink:href="https://wayback.archive-it.org/5151/*/http://www.princeton.edu/president20/" xlink:type="simple"/>
```

**Comment by jrgriffiniii** [comment]
I think that this might be a data cleaning issue, as https://github.com/pulibrary/pulfalight/pull/171 uses disambiguation for IIIF manifests. However, there might still be dead links in these elements.

**Comment by tpendragon** [comment]
Closed as duplicate of #490
